### ISSUE TITLE
[Feature Request]: Separate subtitles by Country Codes.

### ISSUE THREAD

**Issue by rDamascena** [MichaelYochpaz/iSubRip#48]
### Confirmations

- [X] I have checked the issues tab, and couldn't find an existing issue with my feature request.

### Description

When downloading subtitles from a certain country code (`GB`, `US`, `MZ`, `BR` or any other), create a folder with the respective name and save the subtitles in this folder.

`Examples:`

`For the same URL from different country codes:`

`MZ` https://tv.apple.com/mz/movie/puss-in-boots-the-last-wish/umc.cmc.3ysvrfdjp7h6q8dt73t3bq23i

Save in `MZ` folder.

`BR` https://tv.apple.com/br/movie/gato-de-botas-2--o-ultimo-pedido/umc.cmc.3ysvrfdjp7h6q8dt73t3bq23i

Save in `BR` folder.

**Create folders named from a list of `ISO 3166-1 alpha-2` two-letter country codes. Always in capital letters.**

Thereafter, save all downloaded subtitles of the `MZ` country code in the `MZ` folder and `BR` in the `BR` folder, extending this to all other country codes and their respective subtitles.

`Suggestion:` Create a configuration option `organize-by-country-code` or `sort-by-country-code`.

**Comment by rDamascena** [comment]
Yes, `MZ` and `BR` storefronts often correspond to the same subtitles. I'm from Brazil and releases are not always available in my country storefront. That's why, I usually download from others storefronts, which do not have forced subtitles, while the movie is not released. When the movie is released in the Brazilian storefront, I download the subtitles again. In this process, multiple "repeat" files of the same movie are created. This way, it is easier to organize and differentiate files from different storefronts.

**Comment by MichaelYochpaz** [comment]
Both of these storefronts refer to the same playlist (and that's a fact, I've checked).
The reason that the BR store had additional subtitles is probably that the playlist has been updated with additional subtitles, not because it's a different storefront.

If you'll download using both URLs (now or at any time) you'll get the same result.
The cause for the additional subtitles is that the playlist has been updated over time, not that you downloaded from a different storefront.

It's possible to do that, but I won't spend effort on it, as it's useless and misleading.
The storefront you download the subtitles from is meaningless. What actually matters is the playlist that the storefront links to, which can be (and usually is) the same for multiple storefronts.
